core/shared/src/main/scala/org/polyvariant/colorize.scala

```diff
@@ -103,10 +119,23 @@ object colorize {
     def invisible: ColorizedString = colored(_.INVISIBLE)
   }
 
+  private[colorize] sealed trait Color {
+    def prefix: String
```

Comment:
let's deal with this inside `render` (pattern matching on `Color` instead of dispatch to an abstract method)